namespace cpp mjollnir.vigridr

struct GameResult {
  1: required bool won
}
